doc/tutorials/04_agent_versus_non_agent_regret.ipynb

```diff
@@ -0,0 +1,705 @@
+{
```

Comment:
<p>Change 3 to 4</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gambitproject/gambit/pull/642/discussion/#comment-2630560100'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>